CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.1]
+
+### Changed
+* Changed the hyp3_insar_stack_for_ts_analysis.jpynb to make it work with newest MintPy 
```

Comment:
Should update this to be specific about *which* MintPy version we're updating to, I assume v1.5.2?